Gopher-Industries/Team-TaskforceX
# 191-Separate-deprecated-code-into-separate-folder

- Took code from before the start of this trimester and placed in folder "Deprecated-TTFX_T2_2022-Reference Only"

Summary of changes:
- Took code from before the start of this trimester and placed in folder "Deprecated-TTFX_T2_2022-Reference Only"

Testing Instructions:
- Follow our "Testing code changes" guide for advice on what is expected.
- Ensure that every "Deleted" file has been "Recreated" in the "Deprecated-TTFX_T2_2022-Reference Only" folder. 
-  Using this branch check inside the folder: Deprecated-TTFX_T2_2022-Reference Only
![image](https://user-images.githubusercontent.com/67585778/204072269-7ecd9cf8-3f5c-457a-88f0-af0163f392cd.png)
- Ensure that all 24 files / directories are there.
